runtime/Python2/src/antlr4/atn/ATNDeserializer.py

```diff
@@ -21,8 +21,12 @@ def __init__(self, options = None):
         self.stateFactories = None
         self.actionFactories = None
 
-    def deserialize(self, data):
+    def deserialize_str(self, data):
         self.data = [ ord(c) for c in data ]
+        return self.deserialize(data)
+
+    def deserialize(self, data):
+        self.data = data
```

Comment:
Do we need both of these methods? I suggest keeping only the single `deserialize`.

Reply:
probably not but I was worried that something else might use it.  I will delete it and see what happens.